<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_search">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Search field</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Search string</string>
    </property>
   </widget>
   <!-- 
   <widget class="QTextEdit" name="textEdit_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>811</width>
      <height>421</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Result list</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   -->
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Search</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>search_icon.png</normaloff>search_icon.png</iconset>
    </property>
   </widget>
   <!-- <widget class="QPushButton" name="btn_directory_dialog">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>80</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>choose directory</string>
    </property>
   </widget> -->
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="actionhelp"/>
   </widget>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionhelp">
   <property name="text">
    <string>help</string>
   </property>
   <property name="statusTip">
    <string>Show help menu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
